gitlab/data_source_gitlab_user.go

```diff
@@ -0,0 +1,133 @@
+package gitlab
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func dataSourceGitlabUser() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGitlabUserRead,
+
+		Schema: map[string]*schema.Schema{
+			"user_id": {
```

Comment:
Last minute comment related to https://github.com/claranet/terraform-provider-gitlab/pull/5#discussion_r186099140

I'd rather have `id` instead of `user_id` in both datasources.

Reply:
Id is a reserved keyword in terraform, that's why I used user_id.

Reply:
Ah, it is indeed at the root of the schema.

Is that why you add `options` them grouped in an option block originally?

Reply:
The `options` block was originally in #5 in order to avoid the same problem but with the `provider` keyword.

Reply:
That may have been a good reason to keep them grouped.